global_fusion/config/berlin.yaml

```diff
@@ -0,0 +1,13 @@
+mode: 0 ## GNSS positioning mode, 0: single, 1:DGPS/DGNSS, 2: kinematic
+
+frequency: 1 ## number of frequency (1:L1,2:L1+L2,3:L1+L2+L5)
+
+soltype: 2 ## type of solution (0:forward,1:backward,2:combined)
+
+satellites: ## all used gnss-systems
+    - GPS
+    - GLONASS
+    - SP3
```

Comment:
SP3 is just the file type for the so-called "satellite orbit solution", which is a precise information of the orbital position of each GNSS satellite. Since it is not a separate satellite system, it might make sense to create a separate option for it. Something like "precise_ephemeris: true/false".